src/test/lit-html_test.ts

```diff
@@ -451,8 +435,24 @@ suite('lit-html', () => {
         assert.equal(container.innerHTML, '<div>321</div>');
       });
 
+      test('updates arrays 2', () => {
```

Comment:
Personal opinion: test titles like this ought to be more descriptive of what the test is actually doing.

Reply:
fixed